src/components/TestDetailsModal.tsx

```diff
@@ -151,6 +157,11 @@ const TestDetailsModal: React.FunctionComponent<{
     }
   };
 
+  const getIgnoreAreaForSelectionId = (idPassed: string) => {
+    let ignoreAreaToReturn = ignoreAreas.filter((area) => idPassed === area.id)[0];
+    return ignoreAreaToReturn;
```

Comment:
could be replaced with `find` as only one is needed 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
also handle the case if it's `undefined`